server/src/food/handler/FoodItemCategory.handler.ts

```diff
@@ -0,0 +1,66 @@
+import { Request, Response } from "express";
+import { Types } from "mongoose";
+import { DbExtension } from "../../db/interface/db-extension";
+import { FoodItemCategory } from "../index";
+
+export class FoodItemCategoryHandler {
+
+  constructor (private _db: DbExtension) {}
+
+  public getAll = async (req: Request, res: Response): Promise<void> => {
+    try {
+      const result = await this._db.fooditemCategory.getAll();
+      res.status(200).json(result);
+    } catch (error) {
+      res.status(error.statusCode).json(error.forResponse());
+    }
+  }
+
+  public getById = async (req: Request, res: Response): Promise<void> => {
+    try {
+      const id = new Types.ObjectId(req.params.id);
```

Comment:
It's not necessary to create a ObjectId Instance your can simply use the string

Reply:
yes, i know but i have always used ObjectId as type.